support/ruby/collins-client/lib/collins/api/ip_address.rb

```diff
@@ -33,11 +33,12 @@ def ipaddress_update! asset_or_tag, old_address = nil, options = {}
       end
     end
 
-    def ipaddress_delete! asset_or_tag, pool = nil
+    def ipaddress_delete! asset_or_tag, pool = nil, address = nil
```

Comment:
This feels odd, in that if you want to pass an address, you have to pass a nil pool? Feels like a better use of an options hash to me, where pool and address are pushed into the final hash param